plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -445,7 +422,8 @@ public String getPluginVersion(File file) {
         File[] files = refDir.listFiles(fileFilter);
         for (File file : files) {
             String pluginName = FilenameUtils.getBaseName(file.getName());
-            String pluginVersion = getPluginVersion(file);
+            String versionNumber = getPluginVersion(file);
+            VersionNumber pluginVersion = new VersionNumber(versionNumber);
```

Comment:
Could combine the 2 lines to condense variables, just a thought:
```suggestion
            VersionNumber pluginVersion = getPluginVersion(new VersionNumber(versionNumber));
```

Reply:
Fixed in ba2291f.